organizational-structure-overview.md

```diff
@@ -0,0 +1,87 @@
+# OpenSSF Governance & Org Structure
+
+## OpenSSF Charter
+
+The overall governance of the OpenSSF is defined in [the Foundation Charter](https://cdn.platform.linuxfoundation.org/agreements/openssf.pdf) (PDF file). 
+
+
+## Definitions
+
+The OpenSSF is comprised of instances of the following categories of official groups:
+
+- The **Governing Board** (GB) oversees the budget and legal status/structure of the OpenSSF. Board meetings may or may not be open to the public.
+- **Committees** are formed by, and report to, the Board. Broadly speaking, Committees handle non-technical matters as needed by the GB, wherein a named set of people are tasked with handling a specific objective. Like the Board meetings, Committee meetings may or may not be open to the public.
+- The **Technical Advisory Council** (TAC) is responsible for the general success of all Technical Initiatives (defined below). TAC meetings are generally open to the public, with the exception of special sessions. 
+  - TAC membership is limited to seven (7) seats, with alternating elections every year. Contributors to Technical Initiatves may run, and each contributor may vote once in TAC elections. 
+- **Technical Initiatives** are groups formed in support of the overall mission of the OpenSSF. Technical Initiatives include Working Groups (WG), Special Interest Groups (SIG), Projects, and services governed by their own Charters with various objectives, reporting relationships, and funding models described below.
+   - All Technical Initiatives are open groups focused on a technical objective, with transparent proceedings that are open to the public, and vary in their objectives.
+   - Technical Initiatives usually manage one or more GitHub repos of their own, may have separate meetings.
+- A **Project** is a Technical Initiative focused on the development and ongoing support of open source licensed software (source code) and its supporting artifacts (technical documentation, etc.).
+- A **Working Group** (WG) focuses on developing other types of deliverables than software such as guides, specifications, and educational material, or conducting initiatives such as an education outreach. While a WG does not produce open source software as a primary artifact it may launch Projects that do so. A WG may also launch Special Interest Groups (SIG) to perform specific tasks. WGs often include some open source code, or use licensed software, in fulfilment of their Charter.
+- **Special Interest Groups** (SIG) are under the direct governance of their reporting WGs and are bound to achieving a very specific goal. 
+- A **Technical Deliverable** can be an open source licensed software and its supporting artifacts, a specification, a technical guide. This is a technical artefact produced by a Technical Initiative.
+- A **Service** is a Technical Initiative in which software is either built or acquired to support or automate OSSF transactions.
```

Comment:
This is new to me - what current OpenSSF activities would you classify as a "service"? Perhaps that's a way of looking at e.g. https://metrics.openssf.org/ ? If so, perhaps we can simply fold these into "Projects", and say that Projects might also involve deploying an instance (perhaps even the reference instance) of the software the Project is focused on.

Reply:
How does sigstore manage this today? They run some services IIRC

Reply:
Yes, that is done informally in a way that today has no formal oversight by OpenSSF. It's hoped that will change. So we could set this up in anticipation, however I think we need to do more work separately about how a service with uptime expectations will be governed.

Reply:
Yes, lets decrease complexity, lets remove service for now, it is part of project.

Reply:
Off hand, I am aware of two projects that provide public-facing services (sigstore and osv.dev). Running a service, whether or not it carries an explicit SLA, has unique constraints that other Projects (which "merely" release code) do not.

Would it sit better to say that a **Service** isn't distinct from a **Project**, but is instead another form of **Technical Deliverable** that has distinct organizational requirements and needs to be identified as such?

Reply:
+1 on that and treating it as a Technical Deliverable!

Reply:
Sounds fine. Just a note that the OSV Schema is an OSSF Technical Deliverable but osv.dev the service and the code behind it is not.